test/test_utils/ec2.py

```diff
@@ -317,10 +317,36 @@ def execute_ec2_training_performance_test(connection, ecr_uri, test_cmd, region=
     # Make sure we are logged into ECR so we can pull the image
     connection.run(f"$(aws ecr get-login --no-include-email --region {region})", hide=True)
 
-    connection.run(f"{docker_cmd} pull -q {ecr_uri} ", hide=False)
+    connection.run(f"{docker_cmd} pull -q {ecr_uri}")
 
     # Run training command, display benchmark results to console
     connection.run(
         f"{docker_cmd} run -e COMMIT_INFO={os.getenv('CODEBUILD_RESOLVED_SOURCE_VERSION')} -v {container_test_local_dir}:{os.path.join(os.sep, 'test')} {ecr_uri} "
         f"{os.path.join(os.sep, 'bin', 'bash')} -c {test_cmd}"
     )
+
+
+def execute_ec2_inference_performance_test(connection, ecr_uri, test_cmd, region=DEFAULT_REGION):
```

Comment:
I might have missed context: why is this function required separately?

Reply:
The inference contains might not be automatically closed after test finished. I wrote 2 different function, one for triggering benchmark in training containers, this one is for inference ones.